src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs

```diff
@@ -243,7 +243,10 @@ private void AddAssemblies (ZipArchiveEx apk)
 
 			int count = 0;
 			foreach (ITaskItem assembly in ResolvedUserAssemblies) {
-
+				if (string.Compare (assembly.GetMetadata ("AndroidSkipAddToPackage"), bool.TrueString, StringComparison.OrdinalIgnoreCase) == 0) {
```

Comment:
Is this way a little better?
```suggestion
				if (bool.TryParse (assembly.GetMetadata ("AndroidSkipAddToPackage"), out bool value) && value) {
```